packages/opentelemetry-exporter-prometheus/src/PrometheusExporter.ts

```diff
@@ -26,16 +26,17 @@ import * as url from 'url';
 import { ExporterConfig } from './export/types';
 import { PrometheusSerializer } from './PrometheusSerializer';
 import { PrometheusLabelsBatcher } from './PrometheusLabelsBatcher';
-
 export class PrometheusExporter implements MetricExporter {
   static readonly DEFAULT_OPTIONS = {
-    port: 9464,
+    host: process.env.OTEL_EXPORTER_PROMETHEUS_HOST ?? '0.0.0.0',
```

Comment:
Not 100% sure here but using ' 0.0.0.0' here may exclude IPv6. Maybe use `[::]`. Or keep it `undefined` as [::] / 0.0.0.0 are anyway the defaults of node?